### PR TITLE
CT5UP-68: Search the browser specification from command line args, ru…

### PR DESCRIPTION
**Pull request by NataliaMarkaki**
…n build and change version

### CODE REVIEW

File: karma-test-runner/src/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const atsTestEntry = resolve(__dirname, "ats-test-entry.js");
 const utsTestEntry = resolve(__dirname, "uts-test-entry.js");
 
 function retrieveBrowserNameWithCorrectCasing(commandLineArgs) {
-  const selectedBrowser =
+  const selectedBrowser = isBrowserSelectedinArray(commandLineArgs._) ||
```
Comment (briandipalma) [inline karma-test-runner/src/index.js]: Why not move all of the code into the function? The ||'ed code.

Comment (NataliaMarkaki) [inline karma-test-runner/src/index.js]: Sounds good to me. I'll do it right away.

File: karma-test-runner/src/index.js [hunk 2]
```diff
@@ -50,6 +49,12 @@ function retrieveBrowserNameWithCorrectCasing(commandLineArgs) {
   }
 }
 
+function isBrowserSelected(commandLineArgs) {
```
Comment (briandipalma) [inline karma-test-runner/src/index.js]: I wonder if something like the below code wouldn't be clearer?

```javascript
function getSelectedBrowser(commandLineArgs) {
  const browser = commandLineArgs.b || commandLineArgs.browser || "chrome";
  const optionlessArgs = commandLineArgs._;

  if (optionlessArgs.includes("--browser")) {
    browser = optionlessArgs[optionlessArgs.indexOf("--browser") + 1];
  }

  return browser.toLowerCase();
}
```

**Comment by thecapdan** [comment]
👍

**Pull request merged**